docs/aws/audit/sesmonitoring/rules/is_dkim_verified_remediation.mdx

```diff
@@ -1,6 +1,246 @@
+4. **Automate DNS Record Addition**: Optionally, automate the addition of CNAME records if you have programmatic access to your DNS provider.
+
+By following these steps, you can prevent the "DKIM Should Be Verified" misconfiguration in Amazon SES using Python scripts.
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console and open the Amazon SES console at https://console.aws.amazon.com/ses/.
+
+2. In the navigation pane, choose "Email Addresses" under "Identity Management".
+
+3. In the list of email addresses, choose the email address that you want to check the DKIM status for.
+
+4. In the details pane, under "DKIM", check the status. If the status is "verified", then DKIM is properly configured. If the status is "pending verification" or "not started", then DKIM is not verified.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List all the identities: Once the AWS CLI is set up, you can list all the identities (email addresses and domains) that you have verified with Amazon SES. Use the following command to do this:
+   ```
+   aws ses list-identities
+   ```
+   This command will return a list of all the identities.
+
+3. Get DKIM attributes: For each identity, you can check if DKIM is verified or not by using the following command:
+   ```
+   aws ses get-identity-dkim-attributes --identities 'identity_name'
+   ```
+   Replace 'identity_name' with the name of the identity you want to check. This command will return the DKIM attributes for the specified identity.
+
+4. Check DKIM verification status: In the output of the previous command, look for the "DkimVerificationStatus" field. If the value of this field is "Success", it means that DKIM is verified for the identity. If the value is "Pending", it means that the verification is still in progress. If the value is "Failed", it means that the verification has failed.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can do this by running the command `pip install boto3` in your terminal.
+
+2. Import the necessary modules and create a session using your AWS credentials. Here's a basic example:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>